system_strings/sysmes_text.en

```diff
@@ -6654,7 +6742,9 @@ And yet those feelings will never be conveyed.
 }
 [sha:7688259652dad5489cc424b5d0fb6114b7a90c4e]{
 -- 愚者の行いは^時に賢者すら堕落させる。^ましてや歓びを与えるなど。
-愚者の行いは^時に賢者すら堕落させる。^ましてや歓びを与えるなど。
+// @flowchart_descr
+Even the wise may be corrupted by the deeds of fools.
+ Especially when those deeds bring joy.
```

Comment:
It's probably this
>...
>Or worse, they may even bring them joy.

Reply:
Subj/obj a bit confusing as suggested (the wise would bring the deeds joy?). Suggest
> Or worse, made to feel joy.

Reply:
No, like this
True that it was unclear though

>Or worse, those deeds may even bring them (the wise) joy.